content/basics/templates/intrinsic-functions/index.md

```diff
@@ -58,6 +58,35 @@ your template more flexible. Let's convert `AmiID` to variable and pass it to re
           InstanceType: !Ref InstanceType
     ```
 
+#### Ref vs Fn::Sub
+
+The intrinsic function Ref returns the value of the specified parameter or resource. When you are declaring a resource in a template and you need to specify another template resource by name, you can use the Ref to refer to that other resource. In general, Ref returns the name of the resource.
+
+Syntax for JSON template
+```
+{ "Ref" : "logicalName" }
+```
+Syntax for YAML template
+```
+Ref: logicalName
+```
+Syntax for the short form:
+```
+!Ref logicalName
+```
+The intrinsic function Fn::Sub substitutes variables in an input string with values that you specify. In your templates, you can use this function to construct commands or outputs that include values that aren't available until you create or update a stack.
+
+Syntax for JSON template
+```
+{ "Fn::Sub" : [ String, { Var1Name: Var1Value, Var2Name: Var2Value } ] }
+```
+Syntax for YAML template
+```
+Fn::Sub:
+  - String
+  - Var1Name: Var1Value
+    Var2Name: Var2Value
+```
```

Comment:
Comparing Ref and Sub before the Sub section is introduced (introduced in challenge) feels premature imo. Also, this is pretty much direct reference from AWS Documentation which is referenced on the line line 21 and 23. I feel this additional content can be left out, but happy to discus why you feel this may be needed.

Reply:
Solution Architect reported customer being confused between the two in their last immersion day, hence have moved the comparison post introducing them both